conference/models.py

```diff
@@ -309,9 +309,16 @@ def getOrCreateForUser(self, user):
 
 
 ATTENDEEPROFILE_VISIBILITY = Choices(
-    ('p', "PUBLIC", 'Publicly available'),
-    ('m', "PARTICIPANTS_ONLY", 'Visible to EuroPython attendees'),
-    ('x', "PRIVATE", 'Visible only to you'),
+    ("p", "PUBLIC", "Publicly available"),
```

Comment:
This change is not useful in this case. Just add the needed code

Reply:
This code block had inconsistent use of quotes, I just made them all the same.

Reply:
@aramboi sure, but this kind of modification could be done in another commit. just to avoid to review the useless modifications to the bug fix

Reply:
I agree with you and this is the way we should do it given enough resources. At this moment I would rather get this in than create more overhead for the people involved.